policy_test.go

```diff
@@ -1554,7 +1555,7 @@ func TestExecutePolicy(t *testing.T) {
 					data:  "foo",
 				},
 			}}, nil)
-		if err == nil {
+		if IsStaticPolicyDataError(err) || !IsDynamicPolicyDataError(err) {
 			t.Fatalf("Expected an error")
```

Comment:
should the messages be changed accordingly:  expected a dynamic policy data error etc, also below